app/gateway/2.8.x/reference/external-plugins.md

```diff
@@ -630,7 +630,7 @@ CMD ["kong", "docker-start"]
 [go-pluginserver]: https://github.com/Kong/go-pluginserver
 [go-plugins]: https://github.com/Kong/go-plugins
 [go-pdk]: https://github.com/Kong/go-pdk
-[kong-pdk]: https://docs.konghq.com/latest/plugin-development/
+[kong-pdk]: https://docs.konghq.com/gateway/latest/pdk/
```

Comment:
```suggestion
[kong-pdk]: /gateway/{{page.kong_version}}/pdk/
```
Even better, we shouldn't be using `https://docs.konghq.com/` URLs in our site. All URLs should be relative if possible. 

You can use the `{{page.kong_version}}` variable for links to other content in the same doc set and it will link to a page in the same version of the doc instead of the latest one. 

For example, if you have a link to `/gateway/` docs from `/gateway/` docs, use the version variable. If the link comes from /deck/ or /hub/ or anything else that isn't `/gateway/`, use `latest` instead.